core/domain/user_jobs_one_off.py

```diff
@@ -626,3 +628,70 @@ def map(item):
     def reduce(key, values):
         values.sort()
         yield (key, values)
+
+
+class ProfilePictureAuditOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that verifies various aspects of profile_picture_data_url in the
+    UserSettingsModel.
+    """
+
+    @classmethod
+    def enqueue(cls, job_id, additional_job_params=None):
+        # We can raise the number of shards for this job, since it goes only
+        # over one type of entity class.
+        super(ProfilePictureAuditOneOffJob, cls).enqueue(job_id, shard_count=32)
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [user_models.UserSettingsModel]
+
+    @staticmethod
+    def map(model):  # pylint: disable=too-many-return-statements
+        if model.deleted:
+            yield ('DELETED', model.id)
```

Comment:
I suggest you prefix all yielded keys with 'SUCCESS -- ' or 'FAILURE -- ' so that the two cases are clearly delineated.

Reply:
Done.